app/src/components/Navigation.js

```diff
@@ -35,22 +38,26 @@ const Navigation = (props) => {
         }
         {
           showUser &&
-            <Link to="/user" className="flex">
-              <img src={avatar} alt="User Account" className="Navigation__icon--left"/>
-            </Link>
+            <div className="Navigation__user__container">
+              <Link to="/user" className="flex">
+                <img src={user.photoURL || avatar} alt="User Account" className="Navigation__icon--left" />
+              </Link>
+              <p className="Navigation__welcome__mssg">Welcome, {user.displayName || user.email}</p>
```

Comment:
I'm worried this is going to spill out too wide, especially in a view constrained for mobile. But let's iterate on it and get some feedback from our designers, Vickie & Miles.